nginx.conf

```diff
@@ -5,7 +5,7 @@ map $http_x_forwarded_proto $web_proxy_scheme {
 
 map $http_user_agent $limit_bots {
   default "";
-  ~*(GoogleBot|bingbot|YandexBot|mj12bot|Apache-HttpClient|Adsbot|Barkrowler) $binary_remote_addr;
+  ~*(GoogleBot|bingbot|YandexBot|mj12bot|Apache-HttpClient|Adsbot|Barkrowler|FacebookBot|dotbot) $binary_remote_addr;
```

Comment:
I don't think these are case insensitive, so can you add `Googlebot` and `DotBot` as well?